.github/workflows/golangci-lint.yml

```diff
@@ -20,11 +20,11 @@ jobs:
       - name: install Go
         uses: actions/setup-go@v2
```

Comment:
Looks like we can upgrade this to `v3` as well.

Reply:
We used to sync these files from the prometheus repo. But we're slowly converting away from CircleCI to just GH Actions. I'm trying to do the minimal work here to keep these up to date until the full conversion (and return to syncing) is done.